client/src/components/Format/Format.component.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+
+// ---| core |---
+import { cn } from 'tools'
+
+// ---| pages |---
+// ---| screens |---
+// ---| components |---
+import { TextProps } from 'components/Text'
+
+// ---| self |---
+import css from './Format.module.scss'
+import { DEFAULT_SIZE_OPTIONS, DEFAULT_WEIGHT_OPTIONS, SIZE_OPTIONS, WEIGHT_OPTIONS, capitalizeFirstLetter, getMaxOptions } from './Format.tool'
+
+export type FormatValue = string | number | Date | undefined | null
+
+const STRING_FORMAT_MAP = {
+  'uppercase': (v: FormatValue) => String(v).trim().toUpperCase(), // ABC DEF
+  'lowercase': (v: FormatValue) => String(v).trim().toLowerCase(), // abc def
+  'capitalize': (v: FormatValue) => String(v).trim().split(/\s+/).map((v: string) => capitalizeFirstLetter(v)).join(' '), // Abc Def
+  'title': (v: FormatValue) => capitalizeFirstLetter(String(v)), // Abc def
+}
+
+const NUMBER_FORMAT_MAP = {
+  // 123,456,789.98
+  'number': (v: FormatValue) => Intl.NumberFormat('en', { maximumFractionDigits: 2 }).format(Number(v)),
```

Comment:
can we create `toNumber = (v, isInt?: boolean) => Intl.NumberFormat('en', { maximumFractionDigits: isInt ? 0 : 2 }).format(Number(v))`